src/main/cli/cli.c

```diff
@@ -3505,13 +3505,21 @@ static void printMap(dumpFlags_t dumpMask, const rxConfig_t *rxConfig, const rxC
     char buf[16];
     char bufDefault[16];
     uint32_t i;
+    uint8_t rcMapIdx;
+    uint8_t defaultRcMapIdx;
 
     headingStr = cliPrintSectionHeading(dumpMask, false, headingStr);
     for (i = 0; i < RX_MAPPABLE_CHANNEL_COUNT; i++) {
-        buf[rxConfig->rcmap[i]] = rcChannelLetters[i];
+    	rcMapIdx = rxConfig->rcmap[i];
+    	if (rcMapIdx != RCMAP_UNMAPPED_INDEX) {
+            buf[rxConfig->rcmap[i]] = rcChannelLetters[i];
+    	}
         if (defaultRxConfig) {
-            bufDefault[defaultRxConfig->rcmap[i]] = rcChannelLetters[i];
-            equalsDefault = equalsDefault && (rxConfig->rcmap[i] == defaultRxConfig->rcmap[i]);
+        	defaultRcMapIdx = defaultRxConfig->rcmap[i];
+        	if (defaultRcMapIdx != RCMAP_UNMAPPED_INDEX) {
+        		bufDefault[defaultRcMapIdx] = rcChannelLetters[i];
+			}
+            equalsDefault = equalsDefault && (defaultRcMapIdx == rcMapIdx);
```

Comment:
Replace TABS with spaces
```suggestion
        defaultRcMapIdx = defaultRxConfig->rcmap[i];
        if (defaultRcMapIdx != RCMAP_UNMAPPED_INDEX) {
            bufDefault[defaultRcMapIdx] = rcChannelLetters[i];
	}
        equalsDefault = equalsDefault && (defaultRcMapIdx == rcMapIdx);
```